lab_1/main.py

```diff
@@ -3,16 +3,23 @@
 A concordance extraction
 """
 
+import re
+def tokenize():
+    word = ''
+    f = open('data.txt', 'r')
+    for line in f:
+        for word in line.split():
+            for i in line:
+                if i.isalpha():
+                    word += i
+                else:
+                    if word:
+                        print(word.lower())
+
+                        word = ''
+tokenize()
+pass
 
-def tokenize(text: str) -> list:
```

Comment:
Please, leave the description of the function how it is - use the already written template.